### ISSUE TITLE
Very slow startup in linux

### ISSUE THREAD

**Issue by thewh1teagle** [agalwood/Motrix#1307]
When I open Motrix for the first time since the PC booted, 
From the terminal via `motrix` command, or from the GUI, it takes 10 to 30 seconds until it opens.
After I close it completely from the system tray, it opens normally in 2 seconds, 
So it happens only once until I reboot the PC and then it happens again.

OS: `ubuntu 20.04.5 LTS`
Motrix version: `1.6.11` installed via `snap` (classic / stable version)
Snap version: `2.57.2`
[Snap package URL](https://snapcraft.io/motrix)

**Comment by 32bitx64bit** [comment]
Probably a issue with the snap version.

I have it installed via appimage and it starts up pretty fast.

**Comment by agalwood** [comment]
Closing as legacy-v1.

Motrix v2 has replaced the legacy 1.x codebase, so this issue is no longer tracked. If it still reproduces with the latest Motrix v2 beta, please open a new issue with the current Motrix version, OS, reproduction steps, and logs.

Motrix v2 已替换 1.x 旧架构，因此不再跟踪此 issue。若问题仍能在最新 Motrix v2 beta 复现，请重新提交 issue，并附上当前版本、系统、复现步骤和日志。
